internal/runners/update/update.go

```diff
@@ -57,14 +57,19 @@ func (u *Update) Run(params *Params) error {
 	// Check for available update
 	upd, err := u.svc.CheckUpdate(context.Background(), params.Channel, "")
 	if err != nil {
-		return locale.WrapInputError(err, "err_update_fetch", "Could not retrieve update information, please verify that '{{.V0}}' is a valid channel.", params.Channel)
+		return errs.AddTips(locale.WrapError(
+			err, "err_update_fetch",
+			"Could not retrieve update information.",
+		), locale.Tl(
+			"err_tip_update_fetch", "Try again, and/or try restarting State Service",
+		))
 	}
 
-	update := updater.NewUpdate(u.an, updater.NewAvailableUpdateFromGraph(upd))
-	if update.NotNeeded() {
+	update := updater.NewUpdateInstall(u.an, updater.NewAvailableUpdateFromGraph(upd))
+	if !update.IsUseful() {
```

Comment:
I'm sorry to keep nitpicking, but this doesn't tell me anything. Please name this something that explains what it does. Eg. `ShouldInstall()`.

Imo `IsValid()` is fine too. I'm guessing you're avoiding it because you attached a specific meaning of "Does this struct hold valid values" to it, but it could also just mean "Is this valid for use".